### PR TITLE
Get Internet Meme Shop dailies if profitable

### PR DESCRIPTION
**Pull request by Rinn**

### CODE REVIEW

**Comment by horrible-little-slime** [comment]
A similar PR was rejected long ago by Scotch and Phillammon, although I didn't then compare the value of bacon to the saleValue of the item.

File: src/dailies.ts [hunk 1]
```diff
@@ -464,6 +466,24 @@ export function gin(): void {
...
+  const internetMemeShopProperties = {
+    _internetViralVideoBought: $item`viral video`,
+    _internetPlusOneBought: $item`plus one`,
+    _internetGallonOfMilkBought: $item`gallon of milk`,
+    _internetPrintScreenButtonBought: $item`print screen button`,
+    _internetDailyDungeonMalwareBought: $item`daily dungeon malware`,
+  };
+
+  for (const [property, item] of Object.entries(internetMemeShopProperties)) {
+    if (!get<boolean>(property) && baconValue * coinmasterPrice(item) < saleValue(item)) {
```
Comment (horrible-little-slime) [inline src/dailies.ts:481]: should we make sure we have enough BACON to do the transaction, or will this acquire bacon as needed?

Comment (Rinn) [reply src/dailies.ts:481]: acquire figures it out automatically

Comment (Rinn) [reply src/dailies.ts:481]: no, no you were right i just had a bunch of them

**Comment by horrible-little-slime** [comment]
If we potentially mallbuy bacon we want to look at mallPrice, not
saleValue; saleValue underestimates because it accounts for sellbot

**Review by Phillammon — APPROVED** [review]

**Pull request merged**